Replicon.Cryptography.SCrypt/DefaultPasswordHash.cs

```diff
@@ -121,13 +121,17 @@ private static SaltParseException InternalTryParseSalt(string salt, out byte[] s
 
         public bool TryParseSalt(string salt, out byte[] saltBytes, out ulong N, out uint r, out uint p, out uint hashLengthBytes)
         {
+#pragma warning disable CS8601 // Possible null reference assignment.
             var error = InternalTryParseSalt(salt, out saltBytes, out N, out r, out p, out hashLengthBytes);
+#pragma warning restore CS8601 // Possible null reference assignment.
             return error == null;
         }
 
         public void ParseSalt(string salt, out byte[] saltBytes, out ulong N, out uint r, out uint p, out uint hashLengthBytes)
         {
+#pragma warning disable CS8601 // Possible null reference assignment.
```

Comment:
Ditto

Reply:
Already throwing `SaltParseExcpetion`, so removing the #pragma directive